NuGet/NuGetFeedClass.ps1

```diff
@@ -277,7 +277,7 @@ class NuGetFeed {
             if ($excludeVersions -contains $version) {
                 continue
             }
-            if (($select -eq 'Exact' -and [NuGetFeed]::NormalizeVersionStr($nuGetVersionRange) -eq [NuGetFeed]::NormalizeVersionStr($version)) -or ($select -ne 'Exact' -and [NuGetFeed]::IsVersionIncludedInRange($version, $nuGetVersionRange))) {
+            if (($select -eq 'Exact' -and [NuGetFeed]::NormalizeVersionStr($nuGetVersionRange) -eq [NuGetFeed]::NormalizeVersionStr($version)) -or ($select -ne 'Exact' -and [NuGetFeed]::IsVersionIncludedInRange([NuGetFeed]::NormalizeVersionStr($version), [NuGetFeed]::NormalizeVersionStr($nuGetVersionRange)))) {

```

Comment:
I don't think you can use the NormalizeVersionStr on the version range, which can be "[24.4.0.0)" - or?

Reply:
Looking in the IsVersionIncludedInRange function - that one already uses NormalizeVersionStr on the individual version numbers found.

Reply:
I was wondering about that to but the first part of the line also used Normalization on the Range so I didn't look too hard at it.

Reply:
Absolutely correct, I reran my tests to see if I just phantom fixed the issue with this and that seems to be the case. 
I'll abandon this.